internal/proto/scan.go

```diff
@@ -99,6 +102,10 @@ func Scan(b []byte, v interface{}) error {
 	case *bool:
 		*v = len(b) == 1 && b[0] == '1'
 		return nil
+	case *time.Time:
+		var err error
+		*v, err = time.Parse(time.RFC3339, util.BytesToString(b))
```

Comment:
Thanks, but why not `RFC3339Nano`?

Reply:
My bad!
I don't know why but I thought that in Write RRC3339.